packages/web/src/dev/ValidatorsListRow.tsx

```diff
@@ -40,20 +45,61 @@ export interface CeloGroup {
     online: boolean
     uptime: number
     attestation: number
+    claims: string[]
   }>
 }
 
 interface Props {
   group: CeloGroup
   expanded: boolean
 }
+interface State {
+  tooltip?: boolean
+}
+
+class ValidatorsListRow extends React.PureComponent<Props & I18nProps, State> {
+  state = {
+    tooltip: false,
+  }
+  tooltipRef = React.createRef<any>()
+  removeDocumentListener: any
+
+  constructor(...args) {
+    super(...(args as [any]))
+
+    const onDocumentClick = (event) => {
+      if (!this.state.tooltip || !this.tooltipRef.current) {
+        return
+      }
+      if (!this.tooltipRef.current.parentNode.contains(event.target)) {
+        this.setState({ tooltip: false })
+      }
+    }
+
+    document.addEventListener('click', onDocumentClick, false)
+
+    this.removeDocumentListener = () =>
+      document.removeEventListener('click', onDocumentClick, false)
+  }
+
+  componentWillUnmount() {
+    this.removeDocumentListener()
+  }
 
-class ValidatorsListRow extends React.PureComponent<Props & I18nProps> {
   render() {
     const { group, expanded } = this.props
+    const { tooltip } = this.state
+    const stopPropagation = (event: any) => {
+      event.preventDefault()
+      event.stopPropagation()
+    }
+    const toggleTooltip = (event: any) => {
+      stopPropagation(event)
+      this.setState({ tooltip: !tooltip })
+    }
     return (
-      <View key={group.address}>
-        <View style={[styles.tableRow, styles.tableRowCont]}>
+      <div style={tooltip ? { zIndex: 2 } : {}}>
```

Comment:
why the inline style?

Reply:
I moved it with the rest of the styles and crashed the app. Seems styles of a div and `StyleSheet.create` don't work fine.

Reply:
oh yeah thats true.